common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -515,6 +515,11 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "This is the base directory on the target/replica warehouse under which data for "
             + "external tables is stored. This is relative base path and hence prefixed to the source "
             + "external table path on target cluster."),
+    STRICT_MANAGED_TABLES_MIGRARTION_OWNER("strict.managed.tables.migration.owner", "hive",
+        "This is used by upgrade tool to check if a managed table should be converted to external table. If the " +
```

Comment:
upgrade tool is not the right term. It should be HiveStrictManagedMigration tool.